website/src/app/user-service.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { User } from './user';
+import { ApiService } from './api.service';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserServiceService {
+
+  constructor(private api: ApiService) {}
+
+  public addUser(user: User): Observable<User> {
+    return this.api.addUser(user);
```

Comment:
Now all of this does is 1:1 proxy to ApiService, there is no justification for them both to exist.